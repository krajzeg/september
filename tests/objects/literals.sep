fruit := [[
	apples: 2, 
	bananas: 3
]]
print("We have", fruit.apples, "apples and", fruit.bananas, "bananas.")

empty := [[]]
print("We also have an empty object, which means this should be true:", empty.is(Object))
