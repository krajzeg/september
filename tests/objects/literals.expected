We have 2 apples and 3 bananas.
We also have an empty object, which means this should be true: <True>
